ansible/roles/debops.ldap/defaults/main.yml

```diff
@@ -100,7 +100,8 @@ ldap__servers_srv_rr: '{{ q("dig", "_ldap._tcp." + ldap__domain + "./SRV", "flat
 # Fully Qualified Domain Name.
 ldap__servers: '{{ (ldap__servers_srv_rr | selectattr("target", "defined")
                     | map(attribute="target") | map("regex_replace", "\.$","") | list)
-                   if ("NXDOMAIN" not in ldap__servers_srv_rr)
+                   if ("NXDOMAIN" not in ldap__servers_srv_rr and
+                       ldap__servers_srv_rr[0])
```

Comment:
Hmm, that doesn't seem right... Can you show the value of `ldap__server_srv_rr` when the `NODATA` is returned? I would think that this would be either a string or a list, in which case `ldap__server_srv_rr[0]` would hold a True-like value.

Reply:
It's an empty string ;)

```
TASK [debops.ldap : debug] *****************************************************************************************************************
ok: [infra.ciphermail.net] => 
  ldap__servers_srv_rr:
  - ''
```

Reply:
I see. I guess that's how Ansible `dig` lookup works then. Did you check if the `NXDOMAIN` and actually found SRV record cases work with this change as well?

Reply:
I tested this change with NXDOMAIN and NODATA responses, as well as with multiple SRV records. Seems to work just fine.